wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/CommandRobot.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package edu.wpi.first.wpilibj2.command;
+
+import edu.wpi.first.hal.FRCNetComm.tInstances;
+import edu.wpi.first.hal.FRCNetComm.tResourceType;
+import edu.wpi.first.hal.HAL;
+import edu.wpi.first.wpilibj.TimedRobot;
+
+/**
+ * CommandRobot is a wrapper over TimedRobot designed to work well with Command based programming.
+ *
+ * <p>The CommandRobot class is intended to be subclassed by a user creating a robot program.
+ */
+public class CommandRobot extends TimedRobot {
+  /** The CommandScheduler instance. */
+  protected CommandScheduler m_scheduler = CommandScheduler.getInstance();
+
+  /** The Command run during autonomous. */
+  protected Command m_autonomousCommand =
+      Commands.print("Default auto Command override autonomousInit to add your own.");
+
+  /** Constructor for CommandRobot. */
+  protected CommandRobot() {
+    this(kDefaultPeriod);
+  }
+
+  /**
+   * Constructor for CommandRobot.
+   *
+   * @param period Period in seconds.
+   */
+  protected CommandRobot(double period) {
+    super(period);
+    HAL.report(tResourceType.kResourceType_Framework, tInstances.kFramework_CommandControl);
```

Comment:
CommandControl refers to the LabVIEW framework.

Reply:
So this should just report as timed?